inspirehep/dojson/hep/fields/bd20x24x.py

```diff
@@ -32,7 +32,6 @@
 
 
 @hep.over('title_translations', '^242[10_][0_]')
```

Comment:
Mmh. This would depend on some specific dojson enhancement: inveniosoftware/dojson#172 so that we can write the rule that decide whether 245 is translated or not, depending on 041.